jobs/dtd-metrics/src/services/global-store.service.ts

```diff
@@ -92,51 +92,57 @@ export class GlobalStoreService {
       .map(({ data }) => Attribute.pick({ id: true, code: true }).parse(data))
       .toArray()
 
-    function assignMacrocategoryId(data: Partial<Tenant>, macroCategoryId: MacroCategory['id']): MacroCategory['id'] {
-      // If we are in a safe macrocategory, just assign it
-      if (!['7', '8'].includes(macroCategoryId)) return macroCategoryId
-      // If the Tenant is a Region or an Autonomy, assign the "Regioni e Province Autonome" macrocategory id
-      if (REGIONI_E_PROVINCE_AUTONOME.includes(data.externalId!.value)) return '7'
-      // Assign the "Consorzi e associazioni regionali" to all others
-      return '8'
-    }
+    const tenants = await readModel.tenants
+      .find(
+        {
+          'data.attributes': {
+            $elemMatch: {
+              id: {
+                $in: attributes.map(({ id }) => id),
+              },
+            },
```

Comment:
minor, but this could be simplified to
```suggestion
          'data.attributes.id': { $in: attributes.map(({ id }) => id) },
```